src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DependencyGraph.cs

```diff
@@ -170,26 +385,18 @@ public class DependencyGraph
             IEnumerable<DependencyDetail> rootDependencies,
             string repoUri,
             string commit,
-            bool includeToolset,
+            DependencyGraphBuildOptions options,
             bool remote,
             ILogger logger,
             string reposFolder,
             IEnumerable<string> remotesMap,
             string testPath)
         {
-            // Fail fast if darcSettings is null in a remote scenario
-            if (remote && remoteFactory == null)
-            {
-                throw new DarcException("Remote graph build requires a remote factory.");
-            }
+            ValidateBuildOptions(remoteFactory, rootDependencies, repoUri, commit, options, 
+                                 remote, logger, reposFolder, remotesMap, testPath);
```

Comment:
In general we chop (isntead of wrap) and don't align things, this violates both of those style patterns.  The rest of this file uses that pattern from what I can see, we should be consistent.